apps/crn-server/src/controllers/research-outputs.ts

```diff
@@ -209,7 +211,7 @@ export default class ResearchOutputs implements ResearchOutputController {
       // TODO: Remove Boom from the controller layer
       // https://asaphub.atlassian.net/browse/CRN-777
       throw Boom.badRequest<ValidationErrorResponse['data']>(
-        VALIDATION_ERROR_MESSAGE,
+        VALIDATION_ERROR_MESSAGE + JSON.stringify(errors),
```

Comment:
why is this necessary? The `+` notation for joining strings is a little old school plus I am not sure what this actually achieves?